clean/ca/corona_pd.py

```diff
@@ -0,0 +1,125 @@
+import re
+import time
+from pathlib import Path
+from typing import List
+
+from bs4 import BeautifulSoup
+
+from .. import utils
+from ..cache import Cache
+from ..config.corona_pd import download_request_headers, index_request_headers
+
+
+class Site:
+    """Scrape file metadata and download files for the Corona Police Department.
+
+    Attributes:
+        name (str): The official name of the agency
+    """
+
+    name = "Corona Police Department."
+
+    def __init__(
+        self,
+        data_dir: Path = utils.CLEAN_DATA_DIR,
+        cache_dir: Path = utils.CLEAN_CACHE_DIR,
+    ):
+        """Initialize a new instance.
+
+        Args:
+            data_dir (Path): The directory where downstream processed files/data will be saved
+            cache_dir (Path): The directory where files will be cached
+        """
+        self.base_url = "https://www.coronaca.gov/government/departments-divisions/police-department/trust-and-transparency#Records%20and%20Community%20Briefing%20Videos"
+        self.data_dir = data_dir
+        self.cache_dir = cache_dir
+        self.cache = Cache(cache_dir)
+
+    @property
+    def agency_slug(self) -> str:
+        """Construct the agency slug."""
+        # Use module path to construct agency slug, which we'll use downstream
+        mod = Path(__file__)
+        state_postal = mod.parent.stem
+        return f"{state_postal}_{mod.stem}"  # ca_corona_pd
+
+    def scrape_meta(self, throttle=0):
+        # construct a local filename relative to the cache directory - agency slug + page url (ca_corona_pd/trust-and-transparency.html)
+        # download the page (if not already cached)
+        # save the index page url to cache (sensible name)
+        base_name = f"{self.base_url.split('/')[-1].split('#')[0]}.html"
+        filename = f"{self.agency_slug}/{base_name}"
+        self.cache.download(
+            filename,
+            self.base_url,
+            user_agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36",
+            headers=index_request_headers,
+        )
+        metadata = []
+        html = self.cache.read(filename)
+        soup = BeautifulSoup(html, "html.parser")
+        body = soup.find("div", class_="accordion_widget mn-accordion")
+        links = body.find_all("a")
+        for link in links:
+            if "public-records-request" not in link["href"]:
+                case_num = self._get_clean_case_num(link)
+                name = link.string
```

Comment:
Name should be a bit more unique for each asset if possible. I would `split` asset_url on the `/` and grab the last element of the list

Reply:
Hey @newsroomdev we can split asset_url on / but it will work only for document types, for the video records then we will have the youtube id as the name then, but i would say we should keep the name as it is on the website, I am attaching the current meta json let me know what you think.

[ca_corona_pd.json](https://github.com/user-attachments/files/16609076/ca_corona_pd.json)